.github/workflows/cypress.yml

```diff
@@ -36,12 +47,23 @@ jobs:
           echo ::set-output name=DIST::${PWD}/dist
           echo ::set-output name=PACKAGE::${REPO##*/}
 
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
         with:
-          php-version: '7.4'
-          coverage: none
-          tools: composer
+          node-version: 16.x
+
+      - name: Get npm cache directory
```

Comment:
We should update this to use the newer action and the much simpler caching approach.

See https://github.com/bluehost/bluehost-wordpress-plugin/blob/master/.github/workflows/cypress.yml#L50-L54

Reply:
Got it, thanks!